client/packages/common/src/intl/locales/en/common.json

```diff
@@ -162,6 +162,11 @@
   "label.additional-info": "Additional Info",
   "label.address": "Address",
   "label.age": "Age",
+  "label.age-months_one": "{{months}} month",
+  "label.age-months_other": "{{months}} months",
+  "label.age-days_one": "{{days}} day",
+  "label.age-days_other": "{{days}} days",
+  "label.age-months-days": "$t(label.age-months, { \"count\": {{months}}, \"months\": {{months}} }), $t(label.age-days, { \"count\": {{days}}, \"days\": {{days}} })",
```

Comment:
Nested language strings! https://www.i18next.com/translation-function/nesting

I think this is the first time we've done this. The syntax is pretty gnarly, but it seems this is the only way to do a string with more than one pluralisable value within it.

Reply:
Can we just concatenate when used ?

Reply:
In practice that would probably be fine in this case, but we can't be sure that a comma-separated concatenation is correct for all languages. 

For example, in Chinese:

> 3个月15天

Reply:
I guess we could stack them vertically and lose the comma?

Reply:
Or even not have a comma, just space ?

Reply:
or put the separator in the translated string
```
"label.age-months-and": "{{months}} months, ",
"label.age-days_one": "{{days}} day",
```

etc.

then `${t('label.age-months-and', { count: months })}${t('label.age-days, {count: days})}`
or something like that. it's a little easier to read than the nested version